streamlit_web.py

```diff
@@ -209,6 +209,12 @@ def df_preprocess(df):
 
 def frequency():
     """main function for frequency analysis"""
+
+    #Create button to return the description for frequency analyzer
```

Comment:
Please make sure to update the comments in this file. Otherwise, everything looks great!

```suggestion
    # Create expandable container to show the description for frequency analyzer
```